ValidationTweaker.php

```diff
@@ -648,28 +672,41 @@ protected function outputSurveyValidationSkip( $instrument )
 
 
 		// Output JavaScript.
+		addLangToJS( [ 'data_entry_287', 'survey_163', 'data_entry_199',
```

Comment:
are these language codes? in which case it would be nice to have comments with details or link to where they can be found.